week3/1-Tuesday/class/exercise1/example4-Lewis.html

```diff
@@ -0,0 +1,37 @@
+<html>
+    <h1>
+        Lines! So many lines!
+    </h1>
+    <hr />
+    <p>
```

Comment:
I think the exercise wanted you to use breaks here to add space. It's a better choice when there is no paragraph text.